hhs_oauth_server/settings/base.py

```diff
@@ -82,7 +82,7 @@
 
 REST_FRAMEWORK = {
     'DEFAULT_THROTTLE_RATES': {
-        'token': None,
+        'token': env('TOKEN_THROTTLE_RATE', None),
```

Comment:
Can we set the default to something other than None? I want to make sure that the headers are set to some value before the end of the sprint.

Reply:
done :)